lib/web/mage/sticky.js

```diff
@@ -41,8 +44,34 @@ define([
 
             if (!isStatic && this.element.is(':visible')) {
                 offset = $(document).scrollTop() - this.parentOffset;
```

Comment:
why do we need this line of code? Variable offset will be redefined in any case in line 49 or 51. 
Could you please review this logic?